components/AccountOverview/index.tsx

```diff
@@ -267,10 +268,13 @@ const AccountOverview: React.FC<AccountOverviewProps & { refetch: () => Promise<
 
   const getInfoBlock = account => {
     const { type } = account
+    const domain = account?.bit_alias
```

Comment:
Not necessary `?.`, If the account is null, the previous line of code will throw error